ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -5292,6 +5303,16 @@ public synchronized SynchronizedMetaStoreClient getSynchronizedMSC() throws Meta
     return syncMetaStoreClient;
   }
 
+    /**
+   * @return the metastore client for the current thread
+   * @throws MetaException
+   */
+  @LimitedPrivate(value = {"Hive"})
```

Comment:
I think these annotations are more useful for public APIs. Since Hive.java is not a public API you can just use @VisibleForTesting annotation.

Reply:
Sure